app/assets/javascripts/common/reports.js

```diff
@@ -140,99 +140,111 @@ DashboardReports = () => {
       const config = withBaseLineConfig(
         additionalCumulativeDiabetesRegistrationsTrendConfig
       );
-
+    
       config.options.scales = {
-        xAxes: [
-          {
-            stacked: true,
-            display: true,
-            gridLines: {
-              display: false,
-              drawBorder: false,
-            },
-            ticks: {
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 12,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
-            },
+        x: {
+          stacked: true,
+          display: true,
+          grid: {
+            display: false,
+            drawBorder: false,
           },
-        ],
-        yAxes: [
-          {
-            id: "cumulativeDiabetesRegistrations",
-            position: "left",
-            stacked: true,
-            display: true,
-            gridLines: {
-              display: false,
-              drawBorder: false,
-            },
-            ticks: {
-              display: false,
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 10,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
-              stepSize: cumulativeDiabetesRegistrationsYAxis.stepSize,
-              max: cumulativeDiabetesRegistrationsYAxis.max,
-              callback: formatNumberWithCommas
+          ticks: {
+            autoSkip: false,
+            color: colors.darkGrey,
+            font: {
+              size: 12,
+              family: "Roboto",
             },
+            padding: 8,
           },
-          {
-            id: "monthlyDiabetesRegistrations",
-            position: "right",
-            stacked: true,
-            display: true,
-            gridLines: {
-              display: true,
-              drawBorder: false,
-            },
-            ticks: {
-              display: false,
-              autoSkip: false,
-              fontColor: colors.darkGrey,
-              fontSize: 10,
-              fontFamily: "Roboto",
-              padding: 8,
-              min: 0,
-              beginAtZero: true,
-              stepSize: monthlyDiabetesRegistrationsYAxis.stepSize,
-              max: monthlyDiabetesRegistrationsYAxis.max,
-              callback: formatNumberWithCommas
+          beginAtZero: true,
+          min: 0,
+        },
+
+        // temporary fix for hiding y axis
+        y: {
+          display: false,
+        },
+
+        yCumulativeDiabetesRegistrations: {
+          // id: "cumulativeDiabetesRegistrations",
```

Comment:
We can remove

Reply:
I did miss this comment, but this will be removed in the upcoming PR drying the config.